### DISCUSSION TITLE
gba box art

### DISCUSSION THREAD

**Discussion by davemk39** [OnionUI/Onion#1445] [Support]
i use the scraper tool to get the box art for all the gba games and they show as fuzzy images as i scroll through the game list
yet if i scrape the box art individually they show properly
i have not had this issue scraping any other systems.
ive run it a few times and its the same.
anyone else getting this
![DSCN0049](https://github.com/OnionUI/Onion/assets/161377412/760e4f83-eb03-47cb-9e7d-e73bbde29c29)

miyoo mini+
onion 4.3.0

**Comment by oherau** [comment]
I also faced the same issue when batch scrapping a whole system.
As a workaround you can delete+scrap again incorrect covers individually.
